gateway/ln-gateway/proto/gatewaylnrpc.proto

```diff
@@ -96,6 +106,17 @@ message SubscribeInterceptHtlcsResponse {
   // A unique identifier for every intercepted HTLC
   // Used to identify an intercepted HTLC through processing and settlement
   bytes intercepted_htlc_id = 11;
+
+  // Optional CircuitKey that LND needs to complete the HTLC
+  CircuitKey key = 12;
+}
+
+message CircuitKey {
+  // The id of the channel that the is part of this circuit.
+  uint64 chan_id = 1;
+
+  // The index of the incoming htlc in the incoming channel.
+  uint64 htlc_id = 2;
```

Comment:
(@okjodom ): Do we still need unique identifiers for intercepted htlcs? I suspect this can go away if we truly are not tracking intermediate state between intercepting the htlc, processing, and then completion.

Reply:
(@okjodom): Or maybe, we keep an indentifier field for which each LN impl chooses to pass a value. LND might want to store the CircuitKey on that field

Reply:
Yes, we can remove the `intercepted_htlc_id`. 

However, for our CLN extension we still need a way to uniquely identify HTLCs so we can send the preimage to the thread that is blocked waiting for the preimage. For this, we use a pair of (incoming_chan_id, htlc_id). This is also what LND uses to uniquely identify an HTLC (they call it a `CircuitKey`).